extensions/default/src/ViewerLayout/ResizablePanelsHook.tsx

```diff
@@ -270,53 +281,55 @@ const useResizablePanels = (
       return;
     }
     const { width: panelGroupWidth } = resizablePanelGroupElemRef.current.getBoundingClientRect();
-    setRightPanelExpandedWidth((size / 100) * panelGroupWidth - sidePanelExpandedInsideBorderSize);
+    setRightPanelExpandedWidth(
+      (size / 100) * panelGroupWidth - panelGroupDefination.shared.expandedInsideBorderSize
+    );
   }, []);
 
   return [
     {
       expandedWidth: leftPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefination.left.collapsedOffsetWidth,
+      collapsedInsideBorderSize: panelGroupDefination.shared.collapsedInsideBorderSize,
+      collapsedOutsideBorderSize: panelGroupDefination.shared.collapsedOutsideBorderSize,
+      expandedInsideBorderSize: panelGroupDefination.shared.expandedInsideBorderSize,
       onClose: onLeftPanelClose,
       onOpen: onLeftPanelOpen,
     },
     {
       expandedWidth: rightPanelExpandedWidth,
-      collapsedWidth: sidePanelCollapsedWidth,
-      collapsedInsideBorderSize: sidePanelCollapsedInsideBorderSize,
-      collapsedOutsideBorderSize: sidePanelCollapsedOutsideBorderSize,
-      expandedInsideBorderSize: sidePanelExpandedInsideBorderSize,
+      collapsedWidth: panelGroupDefination.right.collapsedOffsetWidth,
+      collapsedInsideBorderSize: panelGroupDefination.shared.collapsedInsideBorderSize,
+      collapsedOutsideBorderSize: panelGroupDefination.shared.collapsedOutsideBorderSize,
+      expandedInsideBorderSize: panelGroupDefination.shared.expandedInsideBorderSize,
       onClose: onRightPanelClose,
       onOpen: onRightPanelOpen,
     },
-    { direction: 'horizontal', id: viewerLayoutResizablePanelGroupId },
+    { direction: 'horizontal', id: panelGroupDefination.groupId },
     {
-      defaultSize: resizablePanelDefaultSize,
-      minSize: resizablePanelDefaultSize,
+      defaultSize: leftResizablePanelDefaultSize,
+      minSize: leftResizablePanelDefaultSize,
       onResize: onLeftPanelResize,
       collapsible: true,
-      collapsedSize: resizablePanelCollapsedSize,
+      collapsedSize: leftRizeablePanelCollapsedSize,
       onCollapse: () => setLeftPanelClosed(true),
       onExpand: () => setLeftPanelClosed(false),
       ref: resizableLeftPanelAPIRef,
       order: 0,
-      id: viewerLayoutResizableLeftPanelId,
+      id: panelGroupDefination.left.panelId,
     },
     { order: 1, id: 'viewerLayoutResizableViewportGridPanel' },
     {
-      defaultSize: resizablePanelDefaultSize,
-      minSize: resizablePanelDefaultSize,
+      defaultSize: rightResizePanelDefaultSize,
```

Comment:
Set this to zero like the left one above.

Reply:
Setting this to 0 throws errors in the console, so I'm going to set it to be the same as `minSize`